Sources/InstantSearchSwiftUI/DataModel/HitsObservableController.swift

```diff
@@ -16,11 +16,12 @@ import InstantSearchTelemetry
   /// HitsController implementation adapted for usage with SwiftUI views
   @available(iOS 13.0, OSX 10.15, tvOS 13.0, watchOS 6.0, *)
   public class HitsObservableController<Hit: Codable>: ObservableObject, HitsController {
-    /// List of hits itemsto present
-    @Published public var hits: [Hit?]
+
+    /// List of hits items to present
+    @Published private(set) public var hits: [Hit?]
```

Comment:
```suggestion
    @Published public var hits: [Hit?]
```
This field is set from the `InstantSearchCore` module, so it cannot be declared as `private(set)` without compilation failure.